<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>431</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>832</width>
    <height>431</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>832</width>
    <height>431</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>300</y>
         <width>821</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">widget_2 {
background-color: yellow
}</string>
       </property>
       <widget class="QPushButton" name="prevButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>60</y>
          <width>41</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/music-player.png</normaloff>:/images/music-player.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="play">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>60</y>
          <width>41</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">play {
background_colot:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/icons/pause.png</normaloff>:/images/icons/pause.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="nextButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>60</y>
          <width>41</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/music-player (1).png</normaloff>:/images/music-player (1).png</iconset>
        </property>
       </widget>
       <widget class="QSlider" name="seekBar">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>441</width>
          <height>25</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="volumeBar">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>70</y>
          <width>121</width>
          <height>25</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Like</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="shuffleButton">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>60</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/shuffle.png</normaloff>:/images/shuffle.png</iconset>
        </property>
       </widget>
       <widget class="QCheckBox" name="repeatButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>60</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>repeat</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>301</height>
        </rect>
       </property>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>351</width>
          <height>311</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QListWidget" name="searchResults">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>311</width>
           <height>241</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>217</width>
           <height>33</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="searchWindow"/>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>0</y>
         <width>401</width>
         <height>291</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="favouritesTab">
        <attribute name="title">
         <string>Favourites</string>
        </attribute>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>100</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>test 1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="createPlaylist">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>90</y>
           <width>100</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Create Playlist</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tracksTab">
        <attribute name="title">
         <string>Tracks</string>
        </attribute>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>391</width>
           <height>291</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
